events/hfe_events_discovery.txt

```diff
@@ -193,7 +429,10 @@ country_event = {
 		if = {
 			limit = {
 				from = {
-					is_country_type = default
+					OR = {
+						is_country_type = default
+						has_origin = hfe_civic_discovery_habitat
```

Comment:
This is wrong

Reply:
Why? It ties to the Origin like it should.